fhir-server/liberty-config/configDropins/defaults/datasource.xml

```diff
@@ -7,7 +7,7 @@
     <!-- ============================================================== -->
     <!-- TENANT: default; DSID: default; TYPE: read-write               -->
     <!-- ============================================================== -->
-    <dataSource id="bootstrapDefaultDefault" jndiName="jdbc/bootstrap_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true" validationTimeout="30s">
+    <dataSource id="bootstrapDefaultDefault" jndiName="jdbc/bootstrap_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true" validationTimeout="30s" isolationLevel="TRANSACTION_READ_COMMITTED">
```

Comment:
do we need to reflect the same with postgres/db2? based on the conversation, it sounds like no since the locks are the way you want.

Reply:
For PostgreSQL default isolation level is Read Committed, which is what we want. For Db2, it's an open question because allegedly the default is Repeatable Read which is more restrictive than necessary, but it doesn't impact us in this case because of the better concurrency control implementation within Db2.